nova_vm/src/ecmascript/syntax_directed_operations/function_definitions.rs

```diff
@@ -62,13 +65,15 @@ pub(crate) fn instantiate_ordinary_function_object(
             env,
             private_env,
         };
+        let f = ordinary_function_create(agent, params);
 
         // 3. Perform SetFunctionName(F, "default").
-        // set_function_name(f, "default");
+        let pk_name = PropertyKey::from_static_str(agent, "default");
```

Comment:
**issue:** We have `"default"` already available as a string in `BUILTIN_STRING_MEMORY`. We should use that instead of trying to reallocate the string.